### ISSUE TITLE
[Bug]: ComfyUI-Studio plugin cant access image resouce

### ISSUE THREAD

**Issue by KarryCharon** [Comfy-Org/desktop#553]
### App Version

v0.4.0

### Expected Behavior

-

### Actual Behavior

-

### Steps to Reproduce

-

### Debug Logs

```powershell
-
```

### Browser Logs

-

### Setting JSON

{
    "Comfy-Desktop.AutoUpdate": true,
    "Comfy-Desktop.SendStatistics": true,
    "Comfy.ColorPalette": "dark",
    "Comfy.UseNewMenu": "Top",
    "Comfy.Workflow.WorkflowTabsPosition": "Topbar",
    "Comfy.Workflow.ShowMissingModelsWarning": true,
    "Comfy.Server.LaunchArgs": {
        "enable-cors-header": "*"
    },
    "AGL.Locale": "zh-CN",
    "LiteGraph.Canvas.MaximumFps": 60,
    "Comfy.DevMode": true,
    "Comfy.Server.ServerConfigValues": {
        "enable-cors-header": "*"
    }
}

### Other

https://github.com/AIGODLIKE/AIGODLIKE-ComfyUI-Studio/issues/67
Image cannot query from local comfyui server. Meet 403 forbidden status code.
We've tried config like `--enable-cors-header *` and `--enable-cors-header "*"`, neither of them worked.
Maybe caused by CORS policy.

┆Issue is synchronized with this [Notion page](https://www.notion.so/Issue-553-Bug-ComfyUI-Studio-plugin-cant-access-image-resouce-1666d73d3650817ab664e1c690531c46) by [Unito](https://www.unito.io)

**Comment by robinjhuang** [comment]
@KarryCharon How does the extension access images? Is it calling a ComfyUI API? Can help look into this

**Comment by KarryCharon** [comment]
> @KarryCharon How does the extension access images? Is it calling a ComfyUI API? Can help look into this

I was made this local pc image path as web StaticResource, so this addon can access image like 
`http://127.0.0.1:8000/Users/xxx/yyy.png?t=1735786023.1082802`

**Comment by KarryCharon** [comment]
> @KarryCharon How does the extension access images? Is it calling a ComfyUI API? Can help look into this

It works fine on MacOS, but get 403 Forbidden on windows

**Comment by KarryCharon** [comment]
> Is this only happening in ComfyUI desktop, or all versions? Happy to take a look at the code, if you can link us to it?
> 
> I have two guesses:
> 
> 1. Something to do with paths - Windows paths may be getting changed / unintentionally escaped somewhere (absolute paths have `:` in them).
> 2. If it's desktop-only, there might be an Electron API that's behaving differently - just need to know what code / extension is listening for that request, so we can take a look.
> 
> p.s. I'd recommend escaping the path and passing it as an escaped string - e.g. `http://127.0.0.1:8000/clear-custom-url/?q=/path/here`. There's no guarantee you won't get name clashes and other easily-avoidable problems with the current format.

Only Desktop, if launch by bat script also works fine!

**Comment by KarryCharon** [comment]
> Is this only happening in ComfyUI desktop, or all versions? Happy to take a look at the code, if you can link us to it?
> 
> I have two guesses:
> 
> 1. Something to do with paths - Windows paths may be getting changed / unintentionally escaped somewhere (absolute paths have `:` in them).
> 2. If it's desktop-only, there might be an Electron API that's behaving differently - just need to know what code / extension is listening for that request, so we can take a look.
> 
> p.s. I'd recommend escaping the path and passing it as an escaped string - e.g. `http://127.0.0.1:8000/clear-custom-url/?q=/path/here`. There's no guarantee you won't get name clashes and other easily-avoidable problems with the current format.

Of course, here the code `https://github.com/AIGODLIKE/AIGODLIKE-ComfyUI-Studio/blob/main/__init__.py`
